app/models/responsible_body.rb

```diff
@@ -88,4 +88,35 @@ def self.with_completed_preorder_info_count
       ",
     )
   end
+
+  def is_ordering_for_schools?
+    schools.that_are_centrally_managed.count.positive?
```

Comment:
Would
```suggestion
    schools.that_are_centrally_managed.any?
```
achieve the same thing?

Reply:
Good spot!